nautobot/docs/models/extras/tag.md

```diff
@@ -16,5 +16,48 @@ The `tag` filter can be specified multiple times to match only objects which hav
 GET /api/dcim/devices/?tag=monitored&tag=deprecated
 ```
 
+Tags can also be created in the ORM of Nautobot. You can create tags and assign the following HEX color values below.
```

Comment:
Are the below options the only permitted color options or can it be free-form?

Reply:
The UI restricts to the provided options, but when bulk-importing tags or editing them via the REST API or ORM, any valid 6-character hex value is permitted. Using a non-standard color will cause some inconsistency when editing the tag via the UI, since the value doesn't correspond to one of the menu options, but otherwise works just fine.

Reply:
@zackt25 Can we add a note to that effect?

Reply:
@bryanculver I've updated the wording of the documentation. I didn't know if I should put a Note at the bottom or include it in the wording above the table.